scripts/test.js

```diff
@@ -61,7 +61,7 @@ var executor = new Executor();
 
 passingTests.forEach(function(passing_test) {
   executor.addCommandlineTest(passing_test)
-      .assertExitCodeOnly();
```

Comment:
Weird spacing issue. Let's revert this part for now.

Reply:
I have tried copy paste from origin but still has this weird space. I think it could be `tab space` thing. Maybe later I will make PR with ESLint that will be responsible with that kind of things.

Reply:
if you want I can create a new PR where it issue will not be